srst2/0.2.0/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:xenial as app
+
+# for easy upgrade later. ARG variables only persist at build time
+ARG SRST2_VER=0.2.0
+
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="SRST2"
+LABEL software.version="v0.2.0"
+LABEL description="Short Read Sequence Typing for Bacterial Pathogens"
+LABEL website="https://github.com/katholt/srst2"
+LABEL license="https://github.com/katholt/srst2/blob/master/LICENSE.txt"
+LABEL maintainer="Holly Halstead"
+LABEL maintainer1.email="holly.halstead@doh.wa.gov"
+
+
+# install dependencies; cleanup apt garbage
+RUN apt-get update && apt-get install -y \
+python2.7 \
+python-pip \
+python-scipy \
+python-biopython \
+make \
+libc6-dev \
+g++ \
+zlib1g-dev \
+build-essential \
+git \
+libx11-dev \
+xutils-dev \
+zlib1g-dev \
+python-pip \
+bowtie2=2.2.6-2 \
```

Comment:
I'm currently torn between leaving `bowtie2=2.2.6-2` on line 33 and `samtools-0.1.18` on lines 41-43 and suggesting that you add the bowtie2 and samtools versions as an `ARG` near the top.

Reasons to leave them as is:
- srst2 requires these specific versions 
- they're unlikely to change in the same ways for the next srst2 update
- the versions are listed in the readme

Reasons to adjust them:
- it's easier to see versions if they're listed near the top
- so that future contributors don't follow similar patterns because their situations are likely different

Reply:
I added versions for bowtie2 and samtools as ARG variables but with a comment separating the main tool and dependency variables to help negate your first point (also in case the versions *do* change) : 

```
# Main package version
ARG SRST2_VER=0.2.0

# Dependency versions
ARG BOWTIE2_VER=2.2.6-2
ARG SAMTOOLS_VER=0.1.18
```